store-and-retrieve-data/use-storagehub-sdk/get-started.md

```diff
@@ -1,4 +1,213 @@
 ---
 title: Get Started
-description: Get up and running quickly with the StorageHub SDK.
+description: TODO
+slug: /store-and-retrieve-data/use-storagehub-sdk/get-started
 ---
+
+# Get Started with the StorageHub SDK
+
+## Overview
+
+The StorageHub SDK is a developer toolkit that makes it easy to build on DataHaven by providing direct access to StorageHub pallet functionalities for managing storage, buckets, and proofs.
+
+The SDK is split into two packages for convenient separation of concerns: `@storagehub-sdk/core` and `@storagehub-sdk/msp-client`. Both work in browser and Node.js environments. You may not always need both at once, so it’s useful to understand the purpose and typical use cases of each:
+
+@storagehub-sdk/core
+
+- Backend‑agnostic building blocks (wallets, EIP‑1193, precompile helpers bridging Substrate↔EVM, Merkle/WASM utilities, HttpClient, shared types).
+- Signing, Merkle/proofs, precompile calls, low‑level HTTP, shared types.
+- This package includes EVM account‑typed helpers, WASM‑backed file utilities, and stable primitives **usable without any backend**.
+- [Read more](https://www.npmjs.com/package/@storagehub-sdk/core){target=_blank}
+
+@storagehub-sdk/msp-client
+
+- Retrieving MSP‑specific client information (health, auth nonce/verify, upload/download endpoints). All MSP‑tied logic lives here.
+- Talking to an MSP backend (auth + file transfer).
+- Includes REST contracts for MSP, token handling, streaming/multipart upload and download helpers.
+- [Read more](https://www.npmjs.com/package/@storagehub-sdk/msp-client){target=_blank}
```

Comment:
Same for this I would move the link to @storagehub-sdk/msp-client
and then remove read more from the bullet list